modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -113,14 +129,28 @@
 
 	subtler_message = span_emote("<b>[user]</b>[space]<i>[user.say_emphasis(subtler_message)]</i>")
 
-	if(istype(target))
+	if(istype(target, /mob))
+		var/mob/M = target
```

Comment:
Why did you do this? It was already checking that `target` was indeed of the type it was declared as, which is `/mob` here.

Plus now you're using a single-letter var, which is definitely no bueno.

Reply:
Because due to the subtler spaghetti code, the "target" var can hold a mob or a number determining view distance that the emote is seen in.
With the addition of this PR, the "target" var might also be a hologram.

Also, originally, there's a show_message proc there that can only be used on mobs, so if I know it's a mob, i'm casting the var to mob type.